D3D12HelloWorld/Graphics/CommandList.cs

```diff
@@ -85,23 +85,20 @@ public sealed class CommandList : IDisposable {
             GetCommandQueue().ExecuteCommandLists(Close());
         }
 
-        public void Reset() {
-            mCommandAllocator.Reset();
-            mCommandList.Reset(mCommandAllocator, null);
-            mIsCommandListClosed = false;
-            SetDescriptorHeaps(mDevice.ShaderVisibleShaderResourceViewAllocator, mDevice.ShaderVisibleSamplerAllocator);
-        }
+        public void Reset(ID3D12CommandAllocator? commandAllocatorOverride = null) {
+            ID3D12CommandAllocator commandAllocator = commandAllocatorOverride ?? mCommandAllocator;
 
-        [Obsolete("Work to refactor off this in favour of Reset() and SetPipelineState(PipelineState), which only uses our own allocator, and which sets the root signature according to the pipeline state.")]
-        public void Reset(ID3D12CommandAllocator commandAllocator, PipelineState pipelineState) {
```

Comment:
This hack has been merged into the DX12GE original `Reset` method by accepting an optional `commandAllocatorOverride`